deps/llvm.mk

```diff
@@ -430,6 +430,7 @@ $(eval $(call LLVM_PATCH,llvm-3.8.0_bindir))
 $(eval $(call LLVM_PATCH,llvm-D14260))
 $(eval $(call LLVM_PATCH,llvm-nodllalias))
 $(eval $(call LLVM_PATCH,llvm-D21271-instcombine-tbaa-3.7))
+$(eval $(call LLVM_PATCH,llvm-win64-reloc-dwarf))
```

Comment:
except on osx where patch is pickier, PR incoming - let me know if the adjusted version works on 3.8 too